### PR TITLE
For Ember Times No. 94: Add Contributor Interview + Contributors

### PR DESCRIPTION
**Pull request by jayjayjpg**
## Sources

Screenshot 2019-04-19 at 16 42 57

### CODE REVIEW

**Review by chrisrng — APPROVED** [review]
✨

File: source/2019-04-12-the-ember-times-issue-94.md [hunk 1]
```diff
@@ -92,65 +93,70 @@ If you'd like to learn more, please take a look at [RFC #463](https://github.com
...
+
+<div class="float-right padded portrait-frame">
+  <img alt="Danielle Adams" title="Danielle Adams - Contributor to Ember" src="/images/blog/emberjstimes/danielleadams.jpg" />
+</div>
+
+In our 10th edition of the contributor interview series, community member **Danielle Adams**, also known as [@danielleadams](https://github.com/danielleadams), talks about her work as a **mentor**, conference **MC** and **public speaker** and shares tips to be present and confident when presenting your ideas. ✨
+
+You can read the full interview on [the Ember Forum](https://discuss.emberjs.com/t/i-contribute-to-ember-with-danielle-adams/16473).
+
+<a class="ember-button ember-button--centered" href="https://discuss.emberjs.com/t/i-contribute-to-ember-with-danielle-adams/16473">Read more</a>
```
Comment (ijlee2) [inline source/2019-04-12-the-ember-times-issue-94.md:127]: It looks like this link is a duplicate of the one right above? (Summary is perfect.)

**Review by Alonski — APPROVED** [review]
Looks great!

File: source/2019-04-12-the-ember-times-issue-94.md (hunk 1 above)
Comment (Alonski) [reply source/2019-04-12-the-ember-times-issue-94.md:127]: Hmm one says Read more and the other is part of the sentence. What do you think @jessica-jordan ?

Comment (jayjayjpg) [reply source/2019-04-12-the-ember-times-issue-94.md:127]: Yes exactly, I always add both a text link and a CTA button to the interview section trying to  get people to click either one of them and actually read the interview:

Screenshot 2019-04-19 at 16 42 57

Comment (ijlee2) [reply source/2019-04-12-the-ember-times-issue-94.md:127]: Ah, I see. Cool.

**Review by ijlee2 — APPROVED** [review]

**Pull request merged**